### PR TITLE
EDN support

### PR DESCRIPTION
**Pull request by gacelita**
Solves #1173 
Adds EDN to the list of possible request types.
image
Enables EDN support in CodeMirror and prettifies the code
image
Adds the `application/edn` mime type
image

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/codemirror/code-editor.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import autobind from 'autobind-decorator';
 import CodeMirror from 'codemirror';
+import 'codemirror/mode/clojure/clojure';
```
Comment (gschier) [inline packages/insomnia-app/app/ui/components/codemirror/code-editor.js]: This should be in `./base-imports` 😄

File: packages/insomnia-app/package.json [hunk 2]
```diff
@@ -152,7 +152,8 @@
     "tough-cookie": "^2.3.1",
     "uuid": "^3.0.0",
     "vkbeautify": "^0.99.1",
-    "whatwg-fetch": "^2.0.1"
+    "whatwg-fetch": "^2.0.1",
+    "zprint-clj": "^0.2.0"
```
Comment (gschier) [inline packages/insomnia-app/package.json]: `package-lock.json` needs to be included in this PR as well. I think NPM 5+ generates `package-lock.json` when `npm install` is run 👍

Comment (JoelSanchez) [reply packages/insomnia-app/package.json]: So, I added the package-lock.json, but it seems to be doing more changes than needed. Should I just commit the line relevant to zprint?

**Comment by JoelSanchez** [comment]
Did a rebase.

**Comment by JoelSanchez** [comment]
Thank you. I think many Clojure devs will turn to Insomnia when/if they need to use an EDN api (starting by my coworkers...)

**Pull request merged**